dumper-integration-tests/redshift-tests/build.gradle

```diff
@@ -0,0 +1,41 @@
+buildscript {
+    repositories {
+        mavenCentral()
+    }
+}
+
+plugins {
+    id 'java'
+    id 'com.adarshr.test-logger' version '3.2.0'
+}
+
+sourceCompatibility = 1.8
+
+repositories {
+    mavenCentral()
+    maven {
+        url "https://redshift-maven-repository.s3-website-us-east-1.amazonaws.com/release"
+    }
+}
+
+dependencies {
+    implementation 'org.codehaus.groovy:groovy-all:3.0.10'
+    implementation 'com.google.guava:guava:31.1-jre'
+    implementation 'org.testng:testng:7.5'
+    implementation 'org.slf4j:slf4j-api:2.0.0-alpha5'
+    implementation 'org.slf4j:slf4j-jdk14:2.0.0-alpha5'
+    implementation 'com.amazon.redshift:redshift-jdbc42:2.1.0.7'
```

Comment:
It looks to me as if this is only ever used at runtime; shouldn't this be `runtimeOnly` instead?